dataclients/kubernetes/kube.go

```diff
@@ -38,8 +38,10 @@ const (
 	defaultIngressClass           = "skipper"
 	ingressRouteIDPrefix          = "kube"
 	defaultEastWestDomainFmt      = "%s.%s.skipper.cluster.local"
-	endpointURIFmt                = "/api/v1/namespaces/%s/endpoints/%s"
-	serviceURIFmt                 = "/api/v1/namespaces/%s/services/%s"
+	endpointsClusterURI           = "/api/v1/endpoints"
+	endpointsNamespaceFmt         = "/api/v1/namespaces/%s/endpoints"
+	servicesClusterURI            = "/api/v1/services"
+	servicesNamespaceFmt          = "/api/v1/namespaces/%s/services"
```

Comment:
Do we need the namespaces versions?

Reply:
There is support for requesting only ingresses in a certain namespace, and it makes sense to do the same for endpoints/services.

Reply:
I just wondered, because of the delete of `namespace`.